xknx/conversion.py

```diff
@@ -0,0 +1,90 @@
+from datetime import time, date, datetime, timedelta
+from .colors import Colors
+from enum import Enum
+
+class xKNXCouldNotConvert(Exception):
+    def __init__(self):
+        pass
+
+    def __str__(self):
+        return "CouldNotConvert"
+
+class DPT():
+    def __init__(self, type = "", name = "", valuerange= (0,100), unit="", bitlength = 1):	
+	    self._type = type
+	    self._name = name
+	    self._valuerange = valuerange
+	    self._unit  = unit
+	    self._bitlength = bitlength
+    @property
+    def type(self):
```

Comment:
Sind solche Getter in Python üblich? Sind doch eh all member frei zugänglich ... Glaube der Python style ist mehr , dass man die member ohne "_" definiert und dann direkt drauf zugreift. Auch wenn ich das als alter C++ Hase grausam finde :-)